src/coord/coord.rs

```diff
@@ -109,134 +118,139 @@ where
     C: comm::Connection,
 {
     pub fn new(config: Config<C>) -> Result<Self, failure::Error> {
-        let mut broadcast_tx = config.switchboard.broadcast_tx(dataflow::BroadcastToken);
```

Comment:
Do we ever assert anywhere in here that the statements are pure?

Reply:
Inside of the coordinator? Confused about where you want such an assertion.

Reply:
Nvm, I was confused about something -- ignore this